backend/src/domain/Games.js

```diff
@@ -0,0 +1,44 @@
+import {Game} from './models/Game';
```

Comment:
Do we need a class for Games? We will only need one instance of Games right?

Reply:
That is true, i just did classes cause java lol. But are there any disadvantages with having a class ?

Reply:
Not sure. Maybe just extra syntax.

Reply:
Oh actually, if we use class we'll have to pass the Games object around? If we just have it as a array in Games.js them maybe it'll be easier to access, since we just need to import that file.

Reply:
hmm ok, we could also keep the class and just export a single games object.